### PR TITLE
Add URL resolving Webhook GraphQL query

### PR DESCRIPTION
**Pull request by pjlast**
Solves #43053 

Builds the webhook external URL using the site config's external URL and the webhook's UUID.

## Test plan

Added unit test

### CODE REVIEW

File: cmd/frontend/graphqlbackend/webhooks_test.go [hunk 1]
```diff
@@ -116,3 +118,63 @@ func TestCreateWebhook(t *testing.T) {
...
+                    ... on Webhook {
+                        id
+                        uuid
+                        url
+                    }
+                }
+			}`
+	schema := mustParseGraphQLSchema(t, db)
+
+	RunTests(t, []*Test{
```
Comment (sashaostrikov) [inline cmd/frontend/graphqlbackend/webhooks_test.go:160]: nit: you can use `RunTest` for a single test case

File: cmd/frontend/graphqlbackend/webhooks.go [hunk 2]
```diff
@@ -33,8 +36,13 @@ func (r *webhookResolver) UUID() string {
 }
 
 func (r *webhookResolver) URL() string {
-	// TODO: Combine external URL and UUID
-	return "TODO"
+	externalURL, err := url.Parse(conf.Get().ExternalURL)
+	if err != nil {
+		// The external URL in the site config should never be invalid
+		return ""
```
Comment (sashaostrikov) [inline cmd/frontend/graphqlbackend/webhooks.go:42]: I know that this should be impossible to get here (URL is parsed before being updated in code host config), but if impossible happens, should we at least log it?

For now this is totally a blind side, silently returning an empty string, which can be troublesome for our clients to debug if they somehow (?!?!) see it

**Review by sashaostrikov — APPROVED** [review]
LGTM! Added a couple of comments

File: cmd/frontend/graphqlbackend/webhooks.go [hunk 3]
```diff
@@ -33,8 +36,13 @@ func (r *webhookResolver) UUID() string {
...
+	}
+	externalURL.Path = fmt.Sprintf("webhooks/%s", r.hook.UUID.String())
```
Comment (ryanslade) [inline cmd/frontend/graphqlbackend/webhooks.go:44]: nit: I think you can use `%v` and just use `r.hook.UUID` since it implements the `Stringer` interface.

**Review by ryanslade — APPROVED** [review]

File: cmd/frontend/graphqlbackend/webhooks.go (hunk 2 above)
Comment (ryanslade) [reply cmd/frontend/graphqlbackend/webhooks.go:42]: Good point, maybe we should return an error in this case actually.

Comment (pjlast) [reply cmd/frontend/graphqlbackend/webhooks.go:42]: Just thought about that as well, going to change it to `(string, error)`

Comment (pjlast) [reply cmd/frontend/graphqlbackend/webhooks.go:42]: Weirdly difficult to get `url.Parse` to fail haha

**Review by sashaostrikov — APPROVED** [review]
:shipit:

**Comment by ryanslade** [comment]
@pjlast Looks like "solved" is not a magic word in GitHub. I think if you used "closes" it would have closed the associated issue too.

**Comment by pjlast** [comment]
@ryanslade thanks, I was getting it mixed up with `resolves` https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

**Pull request merged**